yoti-sdk-api/src/main/java/com/yoti/api/client/DocumentDetails.java

```diff
@@ -6,12 +6,17 @@
  */
 public interface DocumentDetails {
 
+    String DOCUMENT_TYPE_PASSPORT = "PASSPORT";
```

Comment:
they are supposedly constants

Reply:
That is a constant - its declared on an interface (public + final are not required)

Reply:
sorry, hadn't realised it was an interface. My bad :-(